### ISSUE TITLE
Add feature availability for Government on more than just features that have gone GA. 

### ISSUE THREAD

**Issue by awatsky** [MicrosoftDocs/powerbi-docs#4125]
There is very little information related to feature availability for Government clouds outside of this page. Since this page lists planned features. Can we include features that are in Preview as well? Even if it lists as _"The feature isn't available in the environment, and we don't have an estimated time frame for delivery"_, this is still an improvement over the current setup. This will give more information to users that at least these features are not slated for delivery anytime soon. 

It also appears that there are some features that were enabled that are not listed here such as:
Automatic Aggregations; and
Power BI Goals
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d017bc96-49d4-7b63-9394-1739cdf7528d
* Version Independent ID: 1e70d724-121d-feea-471e-307c2f27dd24
* Content: [Power BI for US government customers - Overview - Power BI](https://learn.microsoft.com/en-us/power-bi/enterprise/service-govus-overview)
* Content Source: [powerbi-docs/enterprise/service-govus-overview.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/enterprise/service-govus-overview.md)
* Service: **powerbi**
* Sub-service: **powerbi-admin**
* GitHub Login: @kfollis
* Microsoft Alias: **kfollis**

**Comment by kfollis** [comment]
@awatsky  - thanks for the feedback! I'm assigning this to a writer for review.

**Comment by awatsky** [comment]
@kfollis  @davidiseminger I don't think you understand what I was requesting as the page looks exactly the same. What I am trying to say is there are releases constantly for Power BI and none of the features whether in preview or not ever list Government. What I'm asking is for a list of Power BI features that are either in preview or released whether it is launched for Government. Then each feature can list whether it has a time table or _the feature isn't available in the environment, and we don't have an estimated time frame for delivery_. Additionally I gave two examples of features that are not even listed on the page and they are still not listed.
